apis/apps/v1alpha1/type.go

```diff
@@ -1088,4 +1089,73 @@ type ClusterObjectReference struct {
 	// Specify whether the object must be defined.
 	// +optional
 	Optional *bool `json:"optional,omitempty"`
+
+	// This option defines the behavior when multiple component objects match the specified @CompDef.
+	// If not provided, an error will be raised when handling multiple matches.
+	//
+	// +optional
+	MultipleClusterObjectOption *MultipleClusterObjectOption `json:"multipleClusterObjectOption,omitempty"`
+}
+
+// MultipleClusterObjectOption defines the options for handling multiple cluster objects matched.
+type MultipleClusterObjectOption struct {
+	// If set, each matched component will have its individual variable with its name as the suffix.
+	// This is required when referencing credential variables that cannot be passed by values.
+	//
+	// +optional
+	Individual *bool `json:"individual,omitempty"`
+
+	// If set, the values from all matched components will be combined into a single variable using the specified strategy.
+	//
+	// +optional
+	Combined *MultipleClusterObjectCombinedOption `json:"combined,omitempty"`
```

Comment:
Combined and Individual can be set at the same time?

Reply:
fixed